test/test-applications/aspnet/Samples.WebForms/Account/Login.aspx.cs

```diff
@@ -5,12 +5,35 @@ namespace Samples.WebForms.Account
 {
     public partial class Login : Page
     {
-        protected void Page_Load(object sender, EventArgs e) { }
+        protected void Page_Load(object sender, EventArgs e) 
+        {
+            if (Request.QueryString["shutdown"] == "1")
+            {
+                Shutdown();
+            }
+        }
 
         protected void LogIn(object sender, EventArgs e)
         {
             FailureText.Text = "Invalid username or password.";
             ErrorMessage.Visible = true;
         }
+
+        private void Shutdown()
+        {
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+
+            foreach (var assembly in assemblies)
+            {
+                foreach (var type in assembly.DefinedTypes)
+                {
+                    if (type.Namespace == "Coverlet.Core.Instrumentation.Tracker")
```

Comment:
Is there only one type in this namespace? No need to check type name?

Reply:
I'm guessing the answer will also explain why use use the `foreach` looks instead of `Type.GetType(...)`.

Reply:
The type name is random so I can't directly use `Type.GetType`. Thankfully there's only one type in the namespace.